pkg/db/functions.go

```diff
@@ -96,7 +96,25 @@ WITH results AS (
                 AND timestamp BETWEEN $2 AND $4
         group by prow_jobs.name, prow_jobs.variants
 )
-SELECT *,
+SELECT pj_name,
+	   pj_variants,	
+       previous_passes,
+       previous_failures,
+       previous_runs,
+       previous_infra_fails,
+       current_passes,
+       current_fails,
+       current_runs,
+       current_infra_fails,
+       id,
+       created_at,
+       updated_at,
+       deleted_at,
+       name,
+       release,
+       variants,
+       test_grid_url,
+       kind,
```

Comment:
This was interesting, it broke on fresh db's once I switched back to gorm automigrate.